app/components/common_post_options/reply_option.tsx

```diff
@@ -4,24 +4,24 @@
 import React, {useCallback} from 'react';
 
 import {fetchAndSwitchToThread} from '@actions/remote/thread';
+import {BaseOption} from '@components/common_post_options';
 import {Screens} from '@constants';
 import {useServerUrl} from '@context/server';
 import {t} from '@i18n';
 import {dismissBottomSheet} from '@screens/navigation';
 
-import BaseOption from './base_option';
-
 import type PostModel from '@typings/database/models/servers/post';
 
 type Props = {
     post: PostModel;
+    location?: typeof Screens[keyof typeof Screens];
```

Comment:
Where is this being used / assigned ? I don't see it anywhere

Reply:
It is used to dismiss the bottom sheet based on if it's `Thread Options` screen or `Post Options`

Reply:
Alright but then:
1. Why is thid the only option component with a  location prop
2. I can't see anywhere in this PR where <ReplyOption location='some-location' is set